### PR TITLE
IGAPP-1227: dynamically wait for localhost in e2e tests

### PR DESCRIPTION
**Pull request by sarahsporck**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

**Review by LeandraH — APPROVED** [review]
Looks like a reasonable attempt, fingers crossed

File: e2e-tests/web/wdio.conf.ts [hunk 1]
```diff
@@ -22,27 +23,29 @@ export const config: WebdriverIO.Config = {
...
 
   jasmineOpts: {
-    defaultTimeoutInterval: 300000,
+    defaultTimeoutInterval: 300_000,
   },
 
   onPrepare: async (): Promise<void> => {
     if (process.env.CI) {
-      const startupDelay = 20000
+      const startupDelay = 10_000
```
Comment (sarahsporck) [inline e2e-tests/web/wdio.conf.ts:39]: a delay is still required for web pack to actually start. otherwise the tests will fail with econrefused.

**Comment by f1sh1918** [comment]
hm pipeline still looks the same, even this seem to be a reasonable change :)

**Comment by f1sh1918** [comment]
@sarahsporck 
i added this pr to fix pipeline
https://github.com/digitalfabrik/integreat-app/pull/878

**Review by f1sh1918 — APPROVED** [review]

**Comment by sarahsporck** [comment]
Hmm. Okay. I guess downgrading is also reasonable. I'll investigate further why webpack cannot resolve the version, as it clearly is then related to node.

**Comment by f1sh1918** [comment]
> Hmm. Okay. I guess downgrading is also reasonable. I'll investigate further why webpack cannot resolve the version, as it clearly is then related to node.

If we put more effort into it, I prefer upgrading instead of downgrading :)

**Pull request merged**